docs/docs/manual/running.md

```diff
@@ -316,37 +316,37 @@ Don’t click the “back” button on your browser - it will likely close your
 
 You can rename a project at any time by clicking inside the project title, which will turn into a text field. Project names don’t have to be unique, as OpenRefine organizes them based on a unique identifier behind the scenes. 
 
-“Permalink” allows you to return to a project at a specific view state - that is, with facets and filters applied. The permalink can help you pick up where you left off if you have to close your project while working with facets and filters. It puts view-specific information directly into the URL: clicking on it will load this current-view URL in the existing tab. You can right-click and copy the Permalink URL to copy the current view state to your clipboard, without refreshing the tab you’re using.
+<span class="menuItems">Permalink</span> allows you to return to a project at a specific view state - that is, with facets and filters applied. The permalink can help you pick up where you left off if you have to close your project while working with facets and filters. It puts view-specific information directly into the URL: clicking on it will load this current-view URL in the existing tab. You can right-click and copy the Permalink URL to copy the current view state to your clipboard, without refreshing the tab you’re using.
```

Comment:
```suggestion
<span class="menuItems">Permalink</span> allows you to return to a project at a specific view state - that is, with facets and filters applied. The permalink can help you pick up where you left off if you have to close your project while working with facets and filters. It puts view-specific information directly into the URL: clicking on it will load this current-view URL in the existing tab. You can right-click and copy the Permalink URL to copy the current view state to your clipboard, without refreshing the tab you’re using.

```
Add a line break so that "Open…" in the following paragraph isn't immediately after this last sentence. And since we are no longer talking about Permalink.

Reply:
There is a very strange issue in markdown to HTML. When a line starts in markdown with span,
```bash
<span class="menuItems">Permalink</span> allows you to return to a project at a specific view state -.......
```
**the HTML is** 
```bash
<span class="menuItems">Permalink</span>
" allows you to return to a project at a specific view state -........"
```
![Screenshot 2020-09-12 at 15 34 33](https://user-images.githubusercontent.com/29340368/92996699-ce22f980-f50d-11ea-81bf-f4554eedf0d5.png)


But when line does not start with a span ,
```bash
t<span class="menuItems">Permalink</span> allows you to return to a project at a specific view state -.......
```
**the HTML is** 
```bash
<p>
"t"
<span class="menuItems">Permalink</span> 
"allows you to return to a project at a specific view state -......."
</p>
```
![Screenshot 2020-09-12 at 15 36 31](https://user-images.githubusercontent.com/29340368/92996702-d1b68080-f50d-11ea-9568-1636afbfef4c.png)


### The line is not enclosed in paragraph when a line starts with span, else it's enclosed in span.